cpp/arcticdb/util/storage_lock.hpp

```diff
@@ -116,7 +116,7 @@ class StorageLock {
         OnExit x{[that=this] () {
             that->mutex_.unlock();
         }};
-        if(!ref_key_exists(store) || ttl_expired(store)) {
+        if(!ref_key_exists(store) || !ttl_not_expired(store)) {
```

Comment:
I find double negatives hard to read.

Reply:
If I were refactoring I would probably replace both sides of the || with a single-line method whose name explains what is being checked `if(no_lock_object_present() || lock_object_within_ttl()`